.github/workflows/conformance.yml

```diff
@@ -229,10 +235,13 @@ jobs:
 
     - name: Set up Go
       uses: actions/setup-go@v2
+      with:
+        go-version: '^1.16.6'
```

Comment:
Why do you need to set the go version and not work with the latest?

Reply:
This surprised me, but the actions/setup-go@v2 isn't pulling down the latest version, it is pulling down go1.15.14 linux/amd64 when left unspecified, which caused the `go install` call later to fail. 

The `^1.16.6` syntax should get us any minor version greater than that, and is not fixed to the exact version.